app/vmselect/promql/active_queries.go

```diff
@@ -23,8 +24,12 @@ func ActiveQueriesHandler(w http.ResponseWriter, r *http.Request) {
 	fmt.Fprintf(w, `{"status":"ok","data":[`)
 	for i, aqe := range aqes {
 		d := now.Sub(aqe.startTime)
+		addr := aqe.quotedRemoteAddr
+		if n := strings.IndexByte(aqe.quotedRemoteAddr, ','); n != -1 {
```

Comment:
I don't understand how this change is related to the mentioned [comment](https://github.com/VictoriaMetrics/VictoriaMetrics/pull/4676#issuecomment-1663203424). 
Would you mind adding some explanations or tests? Although, would using of `%q` formatting on L:31 help here?

Reply:
`aqe.quotedRemoteAddr` can be `value` or `value, X-Forwarded-For: value`. 
If you use `%q` formatting, the value will be `remote_addr: "value, X-Forwarded-For: value"`

https://github.com/VictoriaMetrics/VictoriaMetrics/blob/bd8ecfb5518936687c8f24cbc82a72069e50ea6b/lib/httpserver/httpserver.go#L437-L444